spec/swagger/swagger_doc_v1.yaml

```diff
@@ -1141,59 +1243,6 @@
             application/vnd.api+json:
               schema:
                 "$ref": "../v1/error_responses.yaml#/422"
-  "/moves/{move_id}/documents":
```

Comment:
as a general rule, we should be cautious about (appearing to) remove features from v1 - suppliers could get upset; are you certain that this can and should be removed?

Reply:
Im happy to keep this even if it's been removed for the suppliers, as I couldn't find this path anywhere in routes: https://github.com/ministryofjustice/hmpps-book-secure-move-api/blob/master/config/routes.rb#L23

Reply:
Is there a way for us to check what endpoints they are/aren't using? 

I double checked we weren't using it on the FE and it hasn't used it for a while. We now use the preferred JSON:API way of creating resources at the resource endpoint of `/documents`.

Reply:
Looking further, I see that this route was recently been removed in PR https://github.com/ministryofjustice/hmpps-book-secure-move-api/pull/612 and so far, there have been no complaints from suppliers. So I guess we have gotten away with this breaking change this time 😬